native/proto/src/proto/expr.proto

```diff
@@ -80,7 +80,8 @@ message Expr {
     ArrayInsert array_insert = 58;
     MathExpr integral_divide = 59;
     ToPrettyString to_pretty_string = 60;
-    UnaryExpr rand = 61;
+    Rand rand = 61;
+    Randn randn = 62;
```

Comment:
The Rand struct could be used here rather than define an identical Randn struct.

```suggestion
    Rand randn = 62;
```